django/contrib/gis/db/backends/postgis/base.py

```diff
@@ -21,6 +21,13 @@ def __init__(self, *args, **kwargs):
 
     def prepare_database(self):
         super().prepare_database()
-        # Check that postgis extension is installed.
+
         with self.cursor() as cursor:
+            cursor.execute(
+                """SELECT installed_version
+                FROM pg_available_extensions
+                WHERE name ='postgis';"""
+            )
+            if cursor.fetchone():
```

Comment:
I guess this should be `cursor.fetchone()[0]` since in the "not installed" case, this is `(None,)` which evaluates to True.

Reply:
Correct, my bad.